src/js/lib/client/package.js

```diff
@@ -47,6 +59,8 @@ config.build = {
     },
     files: [
         { from: paths.dist.content, to: "." }
+    ,
+        { from: paths.dist.bin, to: '.' },
```

Comment:
is this prettier formatting? If so, we have rather serious problem here

Reply:
No, this is an artefact from removing the initial prettier formatting commit.